src/install/install.zig

```diff
@@ -1458,6 +1375,7 @@ pub const PackageManager = struct {
     resolve_tasks: TaskChannel,
     timestamp_for_manifest_cache_control: u32 = 0,
     extracted_count: u32 = 0,
+    aliasMap: std.ArrayHashMapUnmanaged(PackageID, String, ArrayIdentityContext, false) = .{},
```

Comment:
this should technically be `alias_map` to fit the zig convention of variable names being snake_case though we don't follow that consistently (particularly with globalObject or globalThis where we want to sort of match JavaScript convention)

Reply:
Ah oops - will fix now.

Reply:
Done in https://github.com/oven-sh/bun/pull/1837/commits/1acbdcb40502a341c47a80f9ccfcb3d97c42e7d2